### PR TITLE
Update documentation to be consistent with each other

### PR DESCRIPTION
**Pull request by w2vgd**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -235,7 +239,7 @@ Description:
 Options:
 * `u/new AMOUNT`
 
-    The `new` keyword can only be used on a property without an existing status. AMOUNT is the amount of money that is offered in the Option to Purchase
+    The `new` keyword can only be used on a property without an existing status. `AMOUNT` is the amount of money that is offered in the Option to Purchase
```
Comment (candyhy) [inline docs/UserGuide.md:242]: not sure if its cut off or the significance of `new` keyword is unclear

File: docs/UserGuide.md [hunk 2]
```diff
@@ -394,7 +398,7 @@ Examples:
 
 Finds all appointments containing any of the specified keywords (case-insensitive) and/or with the given parameters. 
 
-Formats:
+Format:
 * `find appointment [n/NAME] [r/REMARKS] [d/DATE]* [t/TIME]*`
 * Note that all options marked with `*` are limited to one per query.
```
Comment (candyhy) [inline docs/UserGuide.md:403]: u mean one input? I think its unclear what one per query refers to

**Review by candyhy — COMMENTED** [review]
LGTM mostly

File: docs/UserGuide.md (hunk 1 above)
Comment (w2vgd) [reply docs/UserGuide.md:242]: The `new` keyword is explaining the `new` in `u/new AMOUNT`. If it is still unclear, then maybe @dvdweien should fix it in his next PR.

File: docs/UserGuide.md (hunk 2 above)
Comment (w2vgd) [reply docs/UserGuide.md:403]: @w-yuchen to fix this. I didn't touch any of their explanations, just fixed the ordering.

**Review by candyhy — APPROVED** [review]
LGTM

**Pull request merged**